backend/src/main/java/com/woowacourse/levellog/prequestion/application/PreQuestionService.java

```diff
@@ -80,9 +97,41 @@ private Member getMember(final Long memberId) {
                 .orElseThrow(() -> new MemberNotFoundException("멤버가 존재하지 않음 [memberId : " + memberId + "]"));
     }
 
-    private void validateLevellog(final PreQuestion preQuestion, final Long levellogId) {
-        if (!preQuestion.isSameLevellog(getLevellog(levellogId))) {
-            throw new InvalidFieldException("입력한 levellogId와 사전 질문의 levellogId가 다릅니다. 입력한 levellogId : " + levellogId);
+    private void validateLevellog(final PreQuestion preQuestion, final Levellog levellog) {
+        if (!preQuestion.isSameLevellog(levellog)) {
+            throw new InvalidFieldException(
+                    "입력한 levellogId와 사전 질문의 levellogId가 다릅니다. 입력한 levellogId : " + levellog.getId());
+        }
+    }
+
+    private void validateExistParticipantByMember(final Team team, final Member member) {
+        final List<Participant> participants = participantRepository.findByTeam(team);
+
+        if (!existsParticipantByMember(participants, member)) {
+            throw new UnauthorizedException("같은 팀에 속한 멤버만 사전 질문을 작성할 수 있습니다.");
+        }
+    }
+
+    private boolean existsParticipantByMember(final List<Participant> participants, final Member member) {
+        return participants.stream()
+                .anyMatch(participant -> participant.getMember().equals(member));
+    }
+
+    private void validateExistPreQuestion(final Levellog levellog, final Member member) {
+        if (!preQuestionRepository.existsByLevellogAndFrom(levellog, member)) {
+            throw new PreQuestionNotFoundException("아직 사전 질문을 작성하지 않았습니다.");
+        }
+    }
+
+    private void validateIsAuthor(final Levellog levellog, final Member member) {
+        if (levellog.isAuthor(member)) {
+            throw new UnauthorizedException("자신의 레벨로그에는 사전 질문을 작성할 수 없습니다.");
+        }
+    }
+
+    private void validateFromMember(final PreQuestion preQuestion, final Member member) {
+        if (!preQuestion.getFrom().equals(member)) {
+            throw new UnauthorizedException("자신의 사전 질문이 아닙니다.");
         }
```

Comment:
getter를 사용하는 것 보다 preQuestion 객체에게 메세지를 보내는 것이 더 명료하겠네요.
```suggestion
    private void validateFromMember(final PreQuestion preQuestion, final Member member) {
        if (preQuestion.isNotSameAuthor(member)) {
            throw new UnauthorizedException("자신의 사전 질문이 아닙니다.");
        }
```

Reply:
```suggestion
    private void validateFromMember(final PreQuestion preQuestion, final Member member) {
        if (!preQuestion.isSameAuthor(member)) {
            throw new UnauthorizedException("자신의 사전 질문이 아닙니다.");
        }
```

boolean 형을 반환하는 isXXX와 같은 메서드는 특별한 상황이 아니면 긍정형으로 만들라고 들었습니다. isNotXX로 만들면 긍정형으로 검증해야할 때 !isNotXXX가 되어 부정에 부정이 되기 때문입니다. 그래서 이 부분을 제안 처럼 수정해봤습니다.